### PR TITLE
Mostly Validator fixes

### PR DESCRIPTION
**Pull request by IhateTrains**
- removed the duplicate Go travelling ambition
- added missing nick_the_boar_acr localisation
- added rapid_advance_tactic localisation
- Validator fixes (title and character history)
- added 2 Wikipedia links
- fixed bracket command in Senate Approval events

### CODE REVIEW

**Comment by loup99** [comment]
Is that because of a bad integration?
- added missing nick_the_boar_acr localisation

File: WTWSMS/history/characters/roman.txt [hunk 1]
```diff
@@ -736,7 +736,7 @@
 	trait="just"
 	trait="skilled_tactician"
 	father=1103000
-	1.1.2={
+	2.11.16={
```
Comment (loup99) [inline WTWSMS/history/characters/roman.txt]: The incorrect date is intentional.

Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: ok

Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: done

**Comment by IhateTrains** [comment]
I've found the localisation in ACR, I will put it in the proper place

**Comment by loup99** [comment]
Okay.

**Comment by IhateTrains** [comment]
done

**Pull request merged**